build/tools/chainlink-fulfilling-requests.md

```diff
@@ -0,0 +1,161 @@
+
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/Oracle.sol";
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/tests/MockOracle.sol";
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/tests/Consumer.sol";
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/tests/BasicConsumer.sol";
+```
+
+Compile the contract and you should see your repository populate
+
+![Compiled contract](../../.github/chainlinkp2-04-Screen_Shot_2021-07-18_at_12.48.49_PM_2.png)
+
+Select the **ChainlinkClient.sol** file, go to line 25 and input the LINK token address:
+
+```jsx
+address constant private LINK_TOKEN_POINTER = 0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846;
+```
+
+Select and compile the **Oracle.sol** file
+
+Define the LINK token contract address, 0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846
+
+and deploy!
+
+Now that we have our Oracle deployed, we will grant our node the ability to fulfill requests to our Oracle. By clicking the **transact** button, call the **setFulfillmentPermission** function with the address of your node, a comma, and the value true:
+
+![setFulfillmentPermission](../../.github/chainlinkp2-05-Screen_Shot_2021-07-18_at_1.16.11_PM.png)
+
+Confirm the transaction in Metamask and record the address of your Oracle as we will need it to create a job for our node.
+
+## Creating jobs
+
+Note: Your node must be actively running. You can re-establish a connection to your active node by running the command:
+
+```bash
+ssh -i $KEY $USER@$REMOTE-IP -L 6688:localhost:6688 -N
+```
+
+If you still can't connect, please refer to the [node deployment article](chainlink-node-deployment.md).
+
+For this tutorial we will use the [EthUnit256 job specification](https://docs.chain.link/docs/fulfilling-requests/). Other Job specifications can be found [here!](https://docs.chain.link/docs/job-specifications/)
+
+Open the Chainlink GUI, which should be at [http://localhost:6688/](http://localhost:6688/) in your web browser and navigate to the **Jobs** tab. There, paste the following into the JSON section with your oracle's contract address:
+
+```json
+{
+  "name": "Get > Uint256",
+  "initiators": [
+    {
+      "type": "runlog",
+      "params": {
+        "address": "YOUR_ORACLE_CONTRACT_ADDRESS"
+      }
+    }
+  ],
+  "tasks": [
+    {
+      "type": "httpget"
+    },
+    {
+      "type": "jsonparse"
+    },
+    {
+      "type": "multiply"
+    },
+    {
+      "type": "ethuint256"
+    },
+    {
+      "type": "ethtx"
+    }
+  ]
+}
+```
+You can also name your job. Click the **Create Job** button, and you should see a confirmation at the top of your screen!
+
+![Job confirmation](../../.github/chainlinkp2-06-Screen_Shot_2021-07-18_at_1.31.17_PM_2.png)
+
+Now our node can begin fulfilling requests!
+
+## Fulfill a request
+
+Note: Changed currentPrice, price and, _price from bytes32 to Uint256
```

Comment:
It is unclear why is this here and what the reader need to do with this information. Could be moved to where this is more relevant?